sycl/test/basic_tests/parallel_for_range.cpp

```diff
@@ -31,119 +31,113 @@ int main() {
   auto DeviceType = D.get_info<info::device::device_type>();
 
   string_class OCLVersionStr = D.get_info<info::device::version>();
-  const bool OCLBackend = (OCLVersionStr.find("OpenCL") != string_class::npos);
-  assert((!OCLBackend || (OCLVersionStr.size() >= 10)) &&
-         "Unexpected device version string"); // strlen("OpenCL X.Y")
-  const char *OCLVersion = &OCLVersionStr[7]; // strlen("OpenCL ")
-
-  // reqd_work_group_size is OpenCL specific.
-  if (OCLBackend) {
```

Comment:
Why is the check for OpenCL backend removed? This change is not connected with PR description and does not look obvious.

Reply:
Ok. I returned the `OCLBackend` back. Implemented as in `sycl/source/detail/error_handling/enqueue_kernel.cpp`:
```cpp
const bool OCLBackend = D.get_platform().get_backend() == backend::opencl;
```